packages/dx-vue-grid-bootstrap4/src/templates/table-stub-header-cell.js

```diff
@@ -0,0 +1,14 @@
+export const TableStubHeaderCell = {
+  props: {
+    tableRow: {},
```

Comment:
Use [recommended way](https://vuejs.org/v2/style-guide/#Prop-definitions-essential) for prop definitions.